### PR TITLE
feat: now button at budget page

### PR DESCRIPTION
**Pull request by UnderKoen**

### CODE REVIEW

**Comment by youngcw** [comment]
test again

**Comment by youngcw** [comment]
moved back to WIP like it was before

**Comment by Jonathan-Fang** [comment]
I agree with @youngcw about making the now button in line with the picker.

Another suggestion is to put the now button under the picker so that the user knows where to look, the Now is kinda sitting with a lot of whitespace around it to me.

File: packages/desktop-client/src/components/common/Link.tsx [hunk 1]
```diff
@@ -25,6 +25,7 @@ type TextLinkProps = {
 
 type ButtonLinkProps = Omit<ComponentProps<typeof Button>, 'variant'> & {
   buttonVariant?: ComponentProps<typeof Button>['variant'];
+  onClick?: MouseEventHandler;
```
Comment (joel-jeremy) [inline packages/desktop-client/src/components/common/Link.tsx:28]: There's already an `onPress` prop to handle button presses

**Comment by youngcw** [comment]
Would it work to use the calendar icon we have, or would that be confusing?

In the dark modes something about my monitor/eyes makes the now button look like its glowing almost like the color changing buttons you see on discord.

**Comment by UnderKoen** [comment]
It definitely looks cleaner! But I would expect a date picker widget when I press it with this icon. What do you think?

image

**Comment by youngcw** [comment]
Isn't it common to have a calendar icon as a "today" button?  I feel like that gets used plenty

**Comment by youngcw** [comment]
Instead of "Now" could the hover text say "Today"?  And I think it would look nicer if the button stayed with the month picker instead of being pinned to the far left header.

**Comment by Jonathan-Fang** [comment]
It looks great! Thanks for all the good work.

One bug I found is that the calendar button when multiple months are displayed is too far to the left; it's not flush with the left side of the month.
image

**Comment by Jonathan-Fang** [comment]
Could a reviewer (maybe @joel-jeremy) help move this PR along?

**Comment by youngcw** [comment]
I do think I would still prefer to have the button stay with the month picker and not get further and further away from the picker when adding months.  Other than that all seems good.

**Comment by youngcw** [comment]
Nice.  I like that a lot better.  If all the check pass then maybe we could sneak this in

**Review by youngcw — APPROVED** [review]

**Comment by youngcw** [comment]
Since this is small, and useful, feel free to merge now.  I unfroze the merge.

**Pull request merged**